{{cookiecutter.project_slug}}/CITATION.cff

```diff
@@ -0,0 +1,8 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: { { cookiecutter.surname } }
+    given-names: { { cookiecutter.first_name } }
+    orcid: https://orcid.org/1234-5678-9101-1121
+title: { { cookiecutter.project_name } }
+doi: xx.yyy/zenodo.zzz
```

Comment:
Should we remove this for now? Also, think it should be `{{cookiecutter.surname}}` syntax instead of current